README.md

```diff
@@ -78,7 +78,13 @@ Recover from a running apiserver (i.e. if the scheduler pods are all down):
 bootkube recover --asset-dir=recovered --kubeconfig=/etc/kubernetes/kubeconfig
 ```
 
-For a complete recovery example please see the [hack/multi-node/bootkube-test-recovery](hack/multi-node/bootkube-test-recovery) script.
+Recover from an etcd backup when self hosted etcd is enabled:
```

Comment:
i think the command line example is already added two line below, right? or you want something different? i will update line 61.